org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/report/HTMLExtension.java

```diff
@@ -203,6 +208,17 @@ public String getFullPathStyle() {
 			theme = theme + "/";
 		String resFile = theme + "css/report.css";
 		
+		// Support to Parse Context Variable
+		if (resFile.contains("@"))
+		{
+			resFile = Env.parseContext(Env.getCtx(), 0, resFile, false);
+			if (Util.isEmpty(resFile))
+			{
+				log.warning("Report theme URL parsing Failed, Defaulting to default report.css url");
```

Comment:
should log the theme expression that fail to be parsed.

Reply:
Added theme value in log.